IanGRap/Squish
# Issue #93 Compactor moves faster over time

Contributors: Alfredo

Important Changes: 
* WallManager.cs: Added a new variable that is multiplied with the compactorTimer every time after every compacting event. Also made very small code changes to clean up the code. 

How to test: 
Go into NewLobbyMatch scene. Try running the game and changing the multiplier values on the WallManager. If you set it to 0.5 is should become twice as fast after every compacting event. If you set it to 2 it will double the timer after every event. By default I set it to 0.8 so that the time between compacting is 80% faster.
